### ISSUE TITLE
Corsair K70 Issues connecting keyboard

### ISSUE THREAD

**Issue by billoverbeck** [SpoinkyNL/Artemis#195]
I found this wonderful app but unfortunately I cannot get my keyboard to pair with the application.

The error I get is:
Couldn't connect keyboard
Check cables and drivers
Enable SDK
Select different keyboard in artemis settings

SO, I've tried all of these to no luck. I've tried both versions of CUE directly downloaded off corsair's website to no avail....

Ive looked for other peoples errors and I've tried restarting my computer, reinstalling artemis/CUE, as well as hitting the off and on switch on artemis a couple times to reestablish a connection with CUE (apparently to no avail).

Any suggestions? I didn't see any other posts with this problem. I'd really like to use this to play Overwatch.

**Comment by SpoinkyNL** [comment]
Hey, sorry to see you're having troubles :c

Could you head into setting and set the logging level to debug and then try to select Corsair again. Then when its told you it couldn't connect, head into Documents > Artemis > logs and get the newest log file and post it here. That way I can see a bit more of what's going wrong

**Comment by billoverbeck** [comment]
Sorry I just found the issue, I needed to install vis c++, found the answer in a closed post (didnt realize I was seeing only open posts!)
link for the lazy for those who find this post
https://www.microsoft.com/en-us/download/confirmation.aspx?id=48145 

And, wow, you responded very quickly! I very much appreciate that. Keep up the fantastic work we all appreciate it. Hopefully things should move forward smoothly with the app. I just reinstalled Artemis so I don't know if I have the logs anymore but I'd assume that is where many people will have problems.

Again, thanks!
